apps-rendering/src/fixtures/item.ts

```diff
@@ -106,7 +106,6 @@ const docFixture = (): Node => {
 
 	doc.appendChild(el);
 
-	// eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- this value is not `null`
 	return doc.firstChild!;
```

Comment:
Do we not lint against prevent [non-null assertions](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)?